app/helpers/application_helper.rb

```diff
@@ -1,2 +1,14 @@
 module ApplicationHelper
+
+  BOOTSTRAP_FLASH_MSG = {
+    success: 'alert-success',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.